gradle.properties

```diff
@@ -1,13 +1,12 @@
-grailsVersion=5.3.5
-grailsGradlePluginVersion=5.3.0
-groovyVersion=3.0.11
-gormVersion=7.3.2
-gormMongoVersion=7.3.0
+grailsVersion=6.2.0
+grailsGradlePluginVersion=6.1.2
+
+gormVersion=8.1.2
+gormMongoVersion=8.2.0
 grailsViewsVersion=2.3.2
```

Comment:
When I last updated grails (I think to grails 5) I had an issue with the latest grails views so deliberately kept it at that version.  I think it might have been around static compilation?  Probably worth a re-look.

Reply:
all good.